Dockerfile.compose

```diff
@@ -2,37 +2,20 @@
 
 FROM debian:stretch-slim
 
-RUN apt-get -y update
+# Install Dependencies
+RUN apt-get update && apt-get install -y \
```

Comment:
This should remain cached for the lifetime of the project. Dependencies will only be updated if you clear out Docker images, or run `docker-compose build` with `--no-cache`.